pkg/vpc/cleanup.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/kris-nova/logger"
 	"github.com/pkg/errors"
 
+	"context"
```

Comment:
I don't know if this was intentionally but the customary way to order imports is:

```
stdlibimports
localmoduleimports
external imports
```

@errordeveloper is this supposed to be different for `eksctl`

(BTW, this is configurable in Goland, in case that's the editor you are using)

Reply:
Yes, I prefer to have it in that sort of order. I do tend to break up external into k8s.io vs anything else. We might have some incosistency, and ordering of the groups should be made more consistent, but we do have `stdlibimports` at the top all the time.

I think VSCode respect this in some way, but I didn't know it can configurable, all IDEs probably use `goimports` (or what was it called?).

Reply:
Good point. This is from `goimports` but apparently it did not group this with the standard library imports because there was a trailing line before the package.